content/en/developers/authorization/oauth2_in_datadog.md

```diff
@@ -16,51 +16,57 @@ further_reading:
 
 This page provides a step-by-step overview on how to implement the OAuth protocol end-to-end on your application once your **confidential** client is created. 
 
-### Implement the OAuth protocol
+## Implement the OAuth protocol
 
 1. Create and configure your OAuth client in the [Developer Platform][16]. 
 
 2. After a user installs your integration, they can click the **Connect Accounts** button to connect their account in the **Configure** tab of the integration tile. 
 
+   {{< img src="developers/authorization/connect-accounts.png" alt="Click the Connect Accounts button" style="width:100%;" >}}
+
    When a user clicks this button, they are directed to the `onboarding_url` that you provided as a part of the OAuth client creation process. This page should be the sign-in page for your platform.
 
 3. Once a user signs in, redirect them to the [OAuth2 Authorization endpoint][6] with the appropriate URL parameters, which includes the added `code_challenge` parameter generated by your application.
 
    To learn how to derive the `code_challenge` parameter, see the [PKCE](#authorization-code-grant-flow-with-pkce) section. Your application must save `code_verifier` for the token request in Step 5.
 
-   - In order to build the URL for this post request, use the `site` query parameter that is provided on the redirect to your `onboarding_url`. 
+   - To build the URL for this GET request, use the `domain` query parameter that is provided on the redirect to your `onboarding_url`. 
    - This parameter is only provided if the user initiates authorization from the Datadog integration tile. See the [Initiate authorization from a third-party location](#Initiate-authorization-from-a-third-party-location) section for more options if the user chooses to initiate authorization externally.  
-   - For example, `site` may be `https://app.datadoghq.com`, `https://app.datadoghq.eu`, `https://us5.datadoghq.com`, or it might have a custom subdomain that must be accounted for, such as `https://<custom_subdomain>.datadoghq.com`. 
-   - To handle multiple Datadog sites dynamically, you can append the Datadog `authorize` path to this constructed URL.
+   - The `domain` query parameter provides the [Datadog site][17] that the authorizing user is in, and is required to construct the URL for this GET request to the Authorize endpoint: `https://api.<domain>/oauth2/v1/authorize?...`.
+
+<div class="alert alert-info">The `domain` parameter was introduced in July 2023, and previous OAuth clients use the `site` parameter for this step. While the `site` parameter is still supported, Datadog recommends using the `domain` parameter for the `Authorize` endpoint. </div>
```

Comment:
Based on our previous conversation, should we say "**Store this value in a secure database or location for later use with additional API endpoints**. "

Reply:
Yeah, I can open a PR to add that